sympy/sets/sets.py

```diff
@@ -940,6 +938,11 @@ def right_open(self):
         """
         return self._args[3]
 
+    @property
+    def is_EmptySet(self):
+        if not (self.end == self.start and (self.left_open or self.right_open)):
+            return (self.end - self.start).is_negative
```

Comment:
The logic here still seems wrong. What does `(a,a)` return? By my read it returns `None` when it should return `True`.